test/integration/offinstall_int_test.go

```diff
@@ -65,22 +61,21 @@ func (suite *OffInstallIntegrationTestSuite) TestInstallAndUninstall() {
 	defaultInstallDir := filepath.Join(defaultInstallParentDir, "IntegrationTest")
 
 	env := []string{
-		"ACTIVESTATE_CLI_DISABLE_RUNTIME=false",
-		"HOME=" + homeDir,
+		constants.DisableRuntime + "=false",
+		constants.IsAdminOverrideEnvVarName + "=false",
+		"VERBOSE=true",
```

Comment:
Just double-checking that HOME isn't needed anymore, as it was removed.

Reply:
Yep. On linux the current home detection doesn't use the HOME env var at all, so this just caused inconsistency in the tests.